### DISCUSSION TITLE
FeathersJS Data & Auth & Live Provider

### DISCUSSION THREAD

**Discussion by omeraplak** [refinedev/refine#3467] [Integration request]
### Is your feature request related to a problem? Please describe.

[FeathersJS](https://docs.feathersjs.com/) is a lightweight web-framework for creating real-time applications and REST APIs using JavaScript or TypeScript. Writing [Data](https://refine.dev/docs/api-reference/core/providers/data-provider/), [Auth](https://refine.dev/docs/api-reference/core/providers/auth-provider/), and [Live](https://refine.dev/docs/api-reference/core/providers/live-provider/) Provider for refine would be great!

### Additional context

It might help to review [Supabase](https://github.com/pankod/refine/tree/next/packages/supabase), [Appwrite](https://github.com/pankod/refine/tree/next/packages/appwrite) packages

### Describe the thing to improve

FeathersJS has a growing community. We think it would be a great pairing with refine 🍕

**Comment by ilomon10** [comment]
it might be great to integrate with feathersjs

**Reply by BatuhanW** [reply]
Hello @ilomon10 would you be interested in contributing to Refine? We'd be more than happy to help you through the process!
